src/packages/recompose/__tests__/branch-test.js

```diff
@@ -49,3 +50,22 @@ test('branch defaults third argument to identity function', t => {
 
   t.is(right, 'Right')
 })
+
+test('branch third argument should not cause console error', t => {
```

Comment:
From one side this test caused this change, from other side this test is not needed now ;-) 
I have no idea what is right solution to remove it now or to allow to stay